packages/gatsby/src/query/__tests__/file-parser.js

```diff
@@ -303,10 +306,11 @@ export const config = async () => {
       ])
     )
 
-    // The second param is a "hint", see: https://jestjs.io/docs/en/expect#tomatchsnapshotpropertymatchers-hint
-    expect(results).toMatchSnapshot({}, `results`)
-    expect(reporter.warn.mock.calls).toMatchSnapshot({}, `warn`)
-    expect(reporter.panicOnBuild.mock.calls).toMatchSnapshot({}, `panicOnBuild`)
+    expect({
+      results,
+      warnCalls: reporter.warn.mock.calls,
+      panicOnBuildCalls: reporter.panicOnBuild.mock.calls,
+    }).toMatchSnapshot()
```

Comment:
this was failing with "obsolete snapshots" even tho jest is supposed to not fail on it when there are snapshots in skipped tests ( https://github.com/facebook/jest/issues/3038 ). Seems like either using multiple snapshots or using the named ones is edge case, so I used single snapshot here instead